### PR TITLE
Solves skills not being deleted on character delete

### PR DESCRIPTION
**Pull request by L3nn0x**
solves #83

---
This change is [Reviewable](https://reviewable.io/reviews/dev-osrose/osirose-new/178)

### CODE REVIEW

File: Database/osirose.sql [hunk 1]
```diff
@@ -236,7 +236,8 @@ CREATE TABLE `skill` (
   `char_id` int(10) unsigned NOT NULL DEFAULT '0',
   `id` smallint(5) unsigned NOT NULL DEFAULT '0',
   `level` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  PRIMARY KEY (`char_id`,`id`)
+  PRIMARY KEY (`char_id`,`id`),
+  CONSTRAINT `char_id` FOREIGN KEY (`char_id`) REFERENCES `characters` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```
Comment (rminderhoud) [inline Database/osirose.sql]: Doesn't it have to be a constraint? Isn't it enough to declare it as foreign key?

Comment (rminderhoud) [reply Database/osirose.sql]: Apparently this is valid syntax but redundant. I will accept and up to you if you want to change it or not

**Review by rminderhoud — DISMISSED** [review]

File: Database/osirose.sql (hunk 1 above)
Comment (L3nn0x) [reply Database/osirose.sql]: It's redundant but used in the whole file, so I'd go for consistency here.

Comment (L3nn0x) [reply Database/osirose.sql]: Actually it poses problems so I'm removing it everywhere.

Comment (RavenX8) [reply Database/osirose.sql]: the reason why it's like that is because that is what the mysql workbench tool made it.

**Comment by rminderhoud** [comment]
LGTM

**Pull request closed without merging**